5-infrastructure/business_unit_1/development/development.auto.example.tfvars

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+parent_folder              = "000000000000"
+boa_gke_project_id         = "prj-bu1-z-boa-gke-xxxx" # 'z' is replaced by {d|n|p} according to environment {development|non-production|production}
+boa_ops_project_id         = "prj-bu1-z-boa-ops-xxxx" # 'z' is replaced by {d|n|p} according to environment {development|non-production|production}
+boa_sec_project_id         = "prj-bu1-z-boa-sec-xxxx" # 'z' is replaced by {d|n|p} according to environment {development|non-production|production}
+boa_sql_project_id         = "prj-bu1-z-boa-sql-xxxx" # 'z' is replaced by {d|n|p} according to environment {development|non-production|production}
+gcp_shared_vpc_project_id  = "prj-z-shared-base-xxxx" # 'z' is replaced by {d|n|p} according to environment {development|non-production|production}
+shared_vpc_name            = "vpc-yyy-shared-base"    # 'yyy' is replaced by {dev|npd|prd} according to environment {development|non-production|production}
+terraform_service_account  = "boa-terraform-z-sa@xxxxxxxxxx.iam.gserviceaccount.com"
+gke_cluster_1_cidr_block   = "000.000.000.000/00"
```

Comment:
Lets only get the project id's and SA email. I think everything else can be defaulted from 3-networks? Users can override this via variable if they change our instructions for 3-networks.

Reply:
shared_vpc_name should be taken as well imo, can be vpc-n-shared-base or vpc-npd-shared-base (Chris) or 
vpc-n-shared-base-spoke (hub&spoke flag mode)

Reply:
we should default to hub and spoke name, users can override if they want